### DISCUSSION TITLE
TESS LC Searching

### DISCUSSION THREAD

**Discussion by nbungart1999** [lightkurve/lightkurve#1098] [Q&A] [answered]
Hello,
First off I just want to say I am very new to this program. But am I doing something wrong when searching for these specific TESS light curves? Every ID I enter returns a could not resolve ID to a sky position. I just wanted to know if I am incorrectly inputting the ID or I have to use a different command or it's just simply the data doesn't exist?
![image](https://user-images.githubusercontent.com/83960306/124303806-deee3b80-db30-11eb-8167-09645ccfc4c2.png)

**Comment by scfleming** [comment, marked as answer]
The resolver relies on Simbad and NED to translate names into coordinates for things like 2MASS IDs.  If Simbad can not resolve your object name, than neither will lightkurve.  For 2MASS, you should include the names like this: "2MASS J16113909+2501000".

If in doubt, try entering your object name here: http://simbad.u-strasbg.fr/simbad/sim-fid.  If a 2MASS ID doesn't resolve there, it won't resolve through lightkurve either.

**Reply by nbungart1999** [reply]
Ok thanks! I'll try that out to see if that works!

**Reply by nbungart1999** [reply]
Ok so I figured out that I shouldn't use the 2MASS ID but rather the TIC ID which I can find from Vizier, so when using lightkurve, is the input just 'TIC 291016342'?

**Comment by scfleming** [comment]
That should work I think.  Not all TIC IDs have light curve data of course.

**Reply by nbungart1999** [reply]
Yeah, I made a mistake thinking one of my systems did but in reality, it said it will be observed later this year
